src/java.management/share/classes/sun/management/ThreadImpl.java

```diff
@@ -525,6 +532,7 @@ private static native void getThreadInfo1(long[] ids,
     private static native void getThreadUserCpuTime1(long[] ids, long[] result);
     private static native long getThreadAllocatedMemory0(long id);
     private static native void getThreadAllocatedMemory1(long[] ids, long[] result);
+    private static native long getThreadAllocatedMemory2();
```

Comment:
We can call this one `getAllThreadAllocatedMemory`, which obviates the need for `2` as the suffix.

Reply:
Fixed.